src/client/Microsoft.Identity.Client/Internal/Broker/IBroker.cs

```diff
@@ -21,6 +21,10 @@ internal interface IBroker
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenSilentParameters acquireTokenSilentParameters);
 
+        Task<MsalTokenResponse> AcquireTokenSilentDefaultUserAsync(
```

Comment:
New method on the IBroker interface to make it explicit

Reply:
what exactly is a default user? what if i don't want to use the default (as the customer signing in), or how is the default set?

---
In reply to: [506400556](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2113#discussion_r506400556) [](ancestors = 506400556)

Reply:
The default user is decided by Windows. I'm trying to get the full details from WAM folks on the exact algorithm. In complex cases, Windows may not be able to determine the "default" user and interaction is required.

Reply:
We could say to simplify "the user logged in in the Windows session" (the account that was used to sign-in to Windows in the morning when you had your coffee mug in the hand?)"?